### PR TITLE
Some changes

### PR DESCRIPTION
**Pull request by user202729**

### CODE REVIEW

File: tex/alg-NT/frobenius.tex [hunk 1]
```diff
@@ -24,7 +30,7 @@ \section{Frobenius elements}
 	Assume $K/\QQ$ is Galois with Galois group $G$.
 	Let $p$ be a rational prime unramified in $K$, and $\kp$ a prime above it.
 	There is a \emph{unique} element $\Frob_\kp \in G$
-	with the property that
+	with the property that, for all $\alpha \in \OO_K$,
```
Comment (user202729) [inline tex/alg-NT/frobenius.tex:33]: For clarity. (actually thinking about it, is "mod 𝔭" well-defined over the whole field K? If K = ℚ[i] then obviously yes but...?)

File: tex/alg-NT/frobenius.tex [hunk 2]
```diff
@@ -338,6 +344,8 @@ \section{Frobenius elements behave well with restriction}
 	\Frob_{\kP} \colon L \to L \]
 and want to know how these are related.
 
+Both maps $\Frob_{\kP}$ and $\Frob_{\kp}$ induce the power-of-$p$ map in the corresponding quotient field, hence we would expect them to be naturally the same.
```
Comment (user202729) [inline tex/alg-NT/frobenius.tex:347]: > thus we would expect the proof of consist of only "typecasting".
Perhaps that's what I mean, but my ability to write isn't that good.

File: tex/alg-NT/frobenius.tex [hunk 3]
```diff
@@ -346,22 +354,13 @@ \section{Frobenius elements behave well with restriction}
...
-%	\end{ques}
-%	Thus $\Frob_{\kP}$ restricted to $\OO_K$ satisfies the
-%	characterizing property of $\Frob_\kp$.
-%\end{proof}
 \begin{proof}
-	TODO: Broken proof. Needs repair.
+	First, $K/\QQ$ is normal, so $\Frob_{\kP}$ fixes the image of $K$, that is,
+	$\Frob_{\kP} \restrict{K} \in \Gal(K/\QQ)$ is well-defined.
+
+	We have the natural map $\phi \colon \OO_K \to \OO_L \to \OO_L/\kP$, and the quotient map $q\colon \OO_K \to \OO_K / \kp$. Since $\kP \subseteq \kP$, then $\phi$ factors through $q$ to give a natural field homomorphism $\OO_K / \kp \to \OO_L / \kP$.
```
Comment (user202729) [inline tex/alg-NT/frobenius.tex:361]: Seem like an interesting side result, but I don't know if this is useful in other ways.

File: tex/alg-NT/ramification.tex [hunk 4]
```diff
@@ -262,22 +262,27 @@ \section{(Optional) Decomposition and inertia groups}
 	How are $\Gal\left( (\OO_K/\kp) / \FF_p \right)$
 	and $\Gal(K/\QQ)$ related?
 \end{quote}
-Absurdly enough, there is an explicit answer:
-\textbf{it's just the stabilizer of $\kp$, at least when
-$p$ is unramified}.
+
```
Comment (user202729) [inline tex/alg-NT/ramification.tex:265]: ("It's natural to consider the stabilizer, not absurd!")

In fact you somewhat derived the result right below (explain how we should consider the stabilizer), I mostly just reorder it to motivate it better.

File: tex/alg-NT/ramification.tex [hunk 5]
```diff
@@ -262,22 +262,27 @@ \section{(Optional) Decomposition and inertia groups}
...
+For this to induce a map in $\Gal\left( (\OO_K/\kp) / \FF_p \right)$, it's necessary that $\sigma(\kp) \subseteq \kp$. So, we consider the subset of automorphisms that fixes $\kp$:
 \begin{definition}
 	Let $D_\kp \subseteq \Gal(K/\QQ)$ be the stabilizer of $\kp$, that is
 	\[ D_\kp \defeq \left\{ \sigma \in \Gal(K/\QQ) \mid \sigma\kp = \kp \right\}. \]
 	We say $D_\kp$ is the \vocab{decomposition group} of $\kp$.
 \end{definition}
-Then, every $\sigma \in D_\kp$ induces an automorphism of $\OO_K / \kp$ by
-\[ \alpha \mapsto \sigma(\alpha) \pmod\kp. \]
+Note that this definition is in fact equivalent to the set of $\sigma$ such that $\sigma(\kp) \subseteq \kp$,
+because a field isomorphism fixes the ideal norm $\Norm(\kp)$.
```
Comment (user202729) [inline tex/alg-NT/ramification.tex:276]: Is there a better proof of this fact?

File: tex/alg-NT/ramification.tex [hunk 6]
```diff
@@ -262,22 +262,27 @@ \section{(Optional) Decomposition and inertia groups}
...
+
 So there's a natural map
 \[ D_\kp \taking\theta \Gal\left( (\OO_K/\kp) / \FF_p \right) \]
 by declaring $\theta(\sigma)$ to just be ``$\sigma \pmod \kp$''.
 The fact that $\sigma \in D_\kp$ (i.e.\ $\sigma$ fixes $\kp$)
 ensures this map is well-defined.
 
+Surprisingly, every element of $\Gal\left( (\OO_K/\kp) / \FF_p \right)$ arises this way from some field automorphism of $K$.
```
Comment (user202729) [inline tex/alg-NT/ramification.tex:284]: Maybe worth saying something about what the kernel mean, but I haven't gotten the intuition yet.

**Comment by vEnhance** [comment]
Looks nice, thanks much!

File: tex/alg-NT/frobenius.tex (hunk 3 above)
Comment (user202729) [reply tex/alg-NT/frobenius.tex:362]: Only after looking at it compiled I realize I made a mistake, it's supposed to say 𝔭 ⊆ 𝔓. (but the more correct thing to say would be 𝔭 ⊆ 𝔓 ∩ 𝒪ₖ ⊆ ker φ, which is the reason why φ factors through q)

**Pull request merged**